docker/images/localdev-server/entrypoint.sh

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+sudo chown localdev /var/run/docker.sock
```

Comment:
Should this username be passed in as ENV VAR? The same way it is passed in as ARG in dockerfile?

Reply:
We'll have to make sure the Dockerfile sets the default value of the ENV var from the arg but I can do that.